### ISSUE TITLE
How to pass values to GenericStringRandomizer with @Randomizer annotation

### ISSUE THREAD

**Issue by ashbike** [benas/random-beans#299]
Thanks for the great work! I really liked this library and planning to use in our project. One question I had was, how to correctly write the following with RandomizerArgument annotation:

```java
// GenericStringRandomizer takes a String [] as argument...
@Randomizer(value = GenericStringRandomizer.class, args = { 
			@RandomizerArgument(value = "Jhon, Doe", type = String[].class) // this is surely not the correct way
})
String firstName;
```
This does not work ofcourse and says Caused by:

```
com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.String[] out of VALUE_STRING token . 
```

I was wondering how to pass a textual representation of String array... any clue?

**Comment by benas** [comment]
Hi @ashbike ,

Thank you for reporting this issue. It is indeed a bug in RB. I pushed a fix in version 3.8.0-SNAPSHOT. Can you give it a try and let me know if it is ok for you? You can use the same syntax as in your example and it should work fine now (cda1de5c94b8753c8aac175cd13eaa7c34ffb9a2 contains a similar example I used to test the fix).

You will need to [add sonatype's snapshot repository](https://github.com/benas/random-beans/wiki/Getting-Started#maven-usage) to be able to import the snapshot version of RB.

Kr,
Mahmoud

**Comment by benas** [comment]
I'm closing this issue, the fix will be part of the upcoming v3.8 release.

@ashbike Please consider giving your feedback with the released version, if you still have any problem, please re-open this issue. Thank you for reporting this out!
